tilequeue/command.py

```diff
@@ -85,10 +84,22 @@ def make_queue(queue_type, queue_name, cfg):
         from tilequeue.queue import MemoryQueue
         return MemoryQueue()
     elif queue_type == 'file':
-        # only support file queues for writing
-        # useful for testing
         from tilequeue.queue import OutputFileQueue
-        fp = open(queue_name, 'w')
+        if os.path.exists(queue_name):
+            assert os.path.isfile(queue_name), \
+                'Could not create file queue. `./{}` is not a file!'.format(
+                    queue_name)
+        else:
+            # Create the file.
+            with open(queue_name, 'w'):
```

Comment:
Is creating an empty file necessary here? Or would the open in append mode take care of that implicitly?